### PR TITLE
prettify demo

### PR DESCRIPTION
**Pull request by notwaldorf**
Bam! I really like those blue/orange buttons! 😍

screen shot 2015-12-08 at 4 14 02 pm
screen shot 2015-12-08 at 4 14 07 pm

### CODE REVIEW

**Comment by cdata** [comment]
Ooh, I like these colors too.

File: bower.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   },
   "devDependencies": {
     "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
```
Comment (cdata) [inline bower.json]: :+1:

**Comment by cdata** [comment]
Our demos are getting so much nicer :laughing: 

LGTM :+1:

**Comment by notwaldorf** [comment]
Thanks!!! 🎉

**Pull request merged**